lottie/meson.build

```diff
@@ -15,6 +15,8 @@ source_file = [
 
 thorvg_lib_dep = declare_dependency(dependencies: thorvg_lib, include_directories: headers)
 
+add_project_link_arguments('-fopenmp', language: 'cpp')
```

Comment:
Could you please add a comment here for later fix?

Reply:
`-fopenmp` option rather brought bug in iOS build. Rather than adding openmp argument, disabled threading option. We didn't verify threading in flutter.